src/Microsoft.DotNet.Tools.Publish/Program.cs

```diff
@@ -34,13 +35,16 @@ public static int Main(string[] args)
 
             app.OnExecute(() =>
             {
-                if (!CheckArg(framework))
+                if (framework.HasValue() && IsUnsupportedFramework(framework.Value()))
```

Comment:
IsUnsupportedFramework only takes a string and return a bool if that string is a valid framework.  But `framework.HasValue()` is to see if the user has specified the `framework` option. I dont want to couple them together.